### PR TITLE
Add blog post about #refugeehackthon

### PR DESCRIPTION
**Pull request by jmewes**

### CODE REVIEW

**Comment by BirgitBader** [comment]
@steffimeier please check from mktng perspective. proof-read will be done afterwards. thanks

File: _posts/2015-10-26-refugeehackathon.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+# Conclusion
+Finally it was a very rich experience for me. I came to know interesting
+people and we had a lot of fun together. Further I learned a number of lessons
+about web development in general and Python/Django in particular.
+
+Hopefully my journey will also result in a little bit of practical help
+for the newcomers.
+
+So many thanks at [@ePages](https://twitter.com/ePages) for sponsoring some work hours,
+the train tickets and hotel costs!
```
Comment (steffimeier) [inline _posts/2015-10-26-refugeehackathon.md]: Just one little remark: ....for sponsoring some work hours and travel costs that I could participate in the hackathon.

File: _posts/2015-10-26-refugeehackathon.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "#refugeehackathon"
+date: "2015-10-26 12:00:00"
+icon: flag
+categories: hackathon
+authors: ["Jan"]
+---
+
+Lots of refugees have landed in Germany. Along with it
```
Comment (JessTinker) [inline _posts/2015-10-26-refugeehackathon.md]: This is text that goes onto the next line as well:

Lots of refugees have landed in Germany. What comes with this is a lot of challenges.

File: _posts/2015-10-26-refugeehackathon.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+there are lots of challenges. Though I am not able to work day and night
```
Comment (JessTinker) [inline _posts/2015-10-26-refugeehackathon.md]: Though I am not able to work day and night

File: _posts/2015-10-26-refugeehackathon.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+for the solution as some people in the crisis management group I wanted to
```
Comment (JessTinker) [inline _posts/2015-10-26-refugeehackathon.md]: for the solution as some people in the crisis management group, I wanted to

File: _posts/2015-10-26-refugeehackathon.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+raise a flag to show that there is a significant amount of people who are willing to help.
+This blog post is a report about my journey to the #refugeehackathon.
+
+# Overview of the event
+The [#refuggeehackathon](https://twitter.com/search?src=typd&q=%23refugeehackathon)
+started on Friday, 24. Oct. with a requirements workshop
```
Comment (JessTinker) [inline _posts/2015-10-26-refugeehackathon.md]: started on Friday, 24th Oct with a requirements workshop

File: _posts/2015-10-26-refugeehackathon.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+where a couple of people visited the refugees and talked about project
+ideas which would be helpful for them. The official start was at Saturday
```
Comment (JessTinker) [inline _posts/2015-10-26-refugeehackathon.md]: The official start was on Saturday

File: _posts/2015-10-26-refugeehackathon.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+morning. After some words of welcome a short video about the challenging
```
Comment (JessTinker) [inline _posts/2015-10-26-refugeehackathon.md]: After some welcoming words, a short video about the challenging

File: _posts/2015-10-26-refugeehackathon.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+situation of refugees traveling through Europe was shown. Then there was a motivational
+speech from a Somalian lady. She invited us not to label people as refugees
+but to regard them as newcomers. Afterwards there was a presentation of the
+project ideas from the workshop and already existing ones.
```
Comment (JessTinker) [inline _posts/2015-10-26-refugeehackathon.md]: project ideas from the workshop as well as outlining existing ideas.

File: _posts/2015-10-26-refugeehackathon.md (hunk 1 above)
Comment (jmewes) [inline _posts/2015-10-26-refugeehackathon.md]: I filled the registration form before the sponsoring was confirmed.

**Pull request merged**